### DISCUSSION TITLE
OnChange event is not working in select dropdown

### DISCUSSION THREAD

**Discussion by purnanallamala** [react-hook-form/react-hook-form#5243] [Q&A]
Hi,

I have created the wrapper component for Select dropdown but having the problem with OnChange event. It is firing. 

Sandbox link:
[https://codesandbox.io/s/react-hook-form-select-onchange-cvhyy?file=/src/App.tsx](https://codesandbox.io/s/react-hook-form-select-onchange-cvhyy?file=/src/App.tsx)

Thanks in advance

**Comment by leapful** [comment]
@purnanallamala

Please describe more detail about your issue and steps to re-produce by following the issue template https://github.com/react-hook-form/react-hook-form/issues/2848

Your CSB sets prop "onChange" on FormSelect component but never use it in react-select component, the "onChange" prop for react-select is from "useController" hook
